CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [29.0.0]
+### Added
+- Reintroduce NotificationController for in-app notifications ([#709](https://github.com/MetaMask/controllers/pull/709))
+- Add token service timeout param and improve tests ([#793](https://github.com/MetaMask/controllers/pull/793))
```

Comment:
Nit: I think it reads better if the test changes aren't referenced. And if we add that it's optional.
```suggestion
- Add optional token service timeout parameter([#793](https://github.com/MetaMask/controllers/pull/793))
```